.github/workflows/releases.yaml

```diff
@@ -0,0 +1,69 @@
+name: Build and Publish Release Binaries
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  release:
+    strategy:
+      matrix:
+        goos: [linux, windows, darwin]
+        goarch: [amd64, arm64]
+        include:
+          - goos: windows
+            goarch: amd64
+            ext: .exe
+          - goos: linux
+            goarch: amd64
+            ext: ""
+          - goos: darwin
+            goarch: amd64
+            ext: ""
+          - goos: linux
+            goarch: arm64
+            ext: ""
+          - goos: darwin
+            goarch: arm64
+            ext: ""
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+          submodules: recursive
+          token: ${{ secrets.GITHUB_TOKEN }}
+    
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: "1.18.x"
+    
+    - name: Install zip
+      run: sudo apt-get install zip
+
+    
+    - name: Build geth binary and create archives
+      env:
+        GOOS: ${{ matrix.goos }}
+        GOARCH: ${{ matrix.goarch }}
+      run: |
+        env GO111MODULE=on go run build/ci.go install ./cmd/geth
+        binary_name="geth-${GOOS}-${GOARCH}${{ matrix.ext }}"
+        mv build/bin/geth "build/bin/${binary_name}"
+        tar -czvf "build/bin/${binary_name}.tar.gz" -C build/bin "${binary_name}"
+        zip -j "build/bin/${binary_name}.zip" "build/bin/${binary_name}"
+        
+    - name: Create and upload release assets
+      uses: softprops/action-gh-release@v1
+      with:
+        draft: false
+        prerelease: false
+        # note you'll typically need to create a personal access token
+        # with permissions to create releases in the other repo
+        token: ${{ secrets.GITHUB_TOKEN }}
+        files: |
+          build/bin/geth-${{ matrix.goos }}-${{ matrix.goarch }}${{ matrix.ext }}
+          build/bin/geth-${{ matrix.goos }}-${{ matrix.goarch }}${{ matrix.ext }}.tar.gz
+          build/bin/geth-${{ matrix.goos }}-${{ matrix.goarch }}${{ matrix.ext }}.zip
```

Comment:
Do we need all 3? I think 1 is probably enough.

Reply:
Wouldn't we want to package release binary so users can easily download it? Or is there no different from just posting the raw binary file?